src/components/cards/DefaultCard/index.tsx

```diff
@@ -3,13 +3,14 @@ import Link from 'next/link'
 
 import { Heading } from '@components/Heading'
 import { Media } from '@components/Media'
+import { Pill } from '@components/Pill'
 import { PlusIcon } from '@root/icons/PlusIcon'
 import { DefaultCardProps } from '../types'
 
 import classes from './index.module.scss'
 
 export const DefaultCard: React.FC<DefaultCardProps> = props => {
-  const { description, href, media, title, className, leader } = props
+  const { adminOnly, description, href, media, title, className, leader } = props
```

Comment:
Instead of taking `adminOnly` this should take `pill` or something. Then you send:

 ```ts
 <Card
   pill={adminOnly ? 'Admin' : undefined}
 />
 ```